crates/re_edit_ui/src/datatype_editors/singleline_string.rs

```diff
@@ -0,0 +1,19 @@
+/// Generic singleline string editor.
+pub fn edit_singleline_string(
+    _ctx: &re_viewer_context::ViewerContext<'_>,
+    ui: &mut egui::Ui,
+    value: &mut impl std::ops::DerefMut<Target = re_types::datatypes::Utf8>,
+) -> egui::Response {
+    edit_singleline_string_impl(ui, value)
+}
+
+/// Non monomorphized implementation of [`edit_singleline_string`].
```

Comment:
We really need to do that everywhere... or rather, the compiler should :unamused: